packages/common/src/store/purchase-content/utils.ts

```diff
@@ -33,6 +33,14 @@ type GetPurchaseSummaryValuesArgs = {
   extraAmount?: number
 }
 
+/**
+ * Gets values for a purchase summary
+ *  - amountDue
+ *  - existingBalance
+ *  - basePrice
+ *  - extraAmount
+ * Throws if number bounds are exceeded
```

Comment:
Is the throw new? This gets called during component rendering, so if it throws, we probably should fix that 😅

Reply:
I think it's actually ok that this throws - it can't accept arbitrary inputs and doing something smarter would probably be to refactor our whole input number system to bigint, etc. or pass through strings. Just added this documentation because the throws is not new!